RULES.md

```diff
@@ -30,6 +30,8 @@ Rules are declared in the [`Notice` module](https://github.com/MobilityData/gtfs
 | [E026](#E026) | Invalid route type | 
 | [E027](#E027) | Missing route short name and long name | 
 | [E028](#E028) | Route long name equals short name | 
+| [E029](#E029) | Missing field `agency_id` for file agency.txt with more than 1 record | 
```

Comment:
Good point for the consistency. 

Also, as you said I'm not sure that adding more detail here is very useful. This could be in the rule description. Modifying this.

Reply:
However, to keep consistency with the GTFS specification, it would be good to format the field names. For example: `E026 - Invalid route type` ->  `E026 - Invalid route_type`.

Reply:
I believe agency_id isn't required if only one agency is defined in agency.txt? In that case, I'd leave the last part of the rule description, otherwise you could interpret the rule differently.